website/docs/docs/running-a-dbt-project/using-the-dbt-ide.md

```diff
@@ -16,23 +16,23 @@ New dbt Cloud accounts will automatically be created with a Development Environm
 
 To create a Development Environment, navigate to the Environments page for your Project and click the "New Environment" button.
 
-<Lightbox src="/img/docs/running-a-dbt-project/0d9f366-Screen_Shot_2019-11-19_at_12.13.28_PM.png" title="Creating a new environment for the Analytics project"/>
+<Lightbox src="/img/docs/running-a-dbt-project/using-the-dbt-ide/empty-env-page.png" title="Creating a new environment for the Analytics project"/>
 
-To create a Development Environment, change the environment "type" to "development". Enter a name like "Dev" for the environment to help identify it amongst your other environments. Finally, click "Save" to finish creating your Development Environment.
+To create a Development Environment, change the environment `Environment Type` to `Development`. Enter a name like "Dev" for the environment to help identify it amongst your other environments. During this step, you can also use a dbt Version. We recommend you selecting the same dbt Version that you're planning to use in `Deployment` environment. Finally, click "Save" to finish creating your Development Environment.
 
-<Lightbox src="/img/docs/running-a-dbt-project/ec04c10-Screen_Shot_2019-11-19_at_12.13.46_PM.png" title="Creating a development environment"/>
+<Lightbox src="/img/docs/running-a-dbt-project/using-the-dbt-ide/create-dev-env.png" title="Creating a development environment"/>
 
 ### Setting up developer credentials
 
 The IDE uses *developer credentials* to connect to your database. These developer credentials should be specific to your user. They should *not* be super user credentials, or the same credentials that you use for your production deployment of dbt.
 
 New dbt Cloud accounts should have developer credentials created automatically as a part of Project creation in the initial application setup.
 
-<Lightbox src="/img/docs/running-a-dbt-project/a810a20-Screen_Shot_2019-11-19_at_11.26.55_AM.png" title="Developer credentials are created during project setup"/>
+<Lightbox src="/img/docs/running-a-dbt-project/using-the-dbt-ide/dev-cred-project-setup.png" title="Developer credentials are created during project setup"/>
 
-Existing accounts *may not* have development credentials configured already. To manage your development credentials, navigate to your [Profile](https://cloud.getdbt.com/#/profile/) in dbt Cloud and find the relevant Project under the **Credentials** header. After entering your developer credentials, you'll be able to access the dbt IDE.
+New users on existing accounts *may not* have development credentials configured already. To manage your development credentials, navigate to your `Settings` in dbt Cloud and under `Your Profile` section, you can find the relevant Project under the **Credentials** header. After entering your developer credentials, you'll be able to access the dbt IDE.
```

Comment:
```suggestion
New users on existing accounts *might not* have their development credentials already configured. To manage your development credentials, go to the [Credentials](https://cloud.getdbt.com/next/settings/profile#credentials) section. Select the relevant project in the list. After entering your developer credentials, you'll be able to access the dbt IDE.
```

Reply:
found a link to the Credentials section so sugg using that to make the navigation steps a lot more simple